airflow-core/tests/unit/executors/test_local_executor.py

```diff
@@ -158,16 +173,73 @@ def test_clean_stop_on_signal(self):
         executor = LocalExecutor(parallelism=2)
         executor.start()
 
-        # We want to ensure we start a worker process, as we now only create them on demand
-        executor._spawn_worker()
-
         try:
             os.kill(os.getpid(), signal.SIGINT)
         except KeyboardInterrupt:
             pass
         finally:
             executor.end()
 
+    @skip_spawn_mp_start
+    def test_worker_process_revive(self):
```

Comment:
What is this test covering? A really process can never revive can it?

Reply:
That's right! I intend to test that the system respawns the same number that died. So the name should probably be updated.